File: src/main/java/be/superjoran/mint/services/PersonService.java
package be.superjoran.mint.services;

import be.superjoran.common.model.DomainObjectCrudService;
import be.superjoran.mint.domain.Person;
import be.superjoran.mint.domain.searchresults.PersonCandidate;

public interface PersonService extends DomainObjectCrudService<Person> {
    Person logIn(PersonCandidate person);

    Person create(PersonCandidate personCandidate);
}

File: src/main/java/be/superjoran/mint/services/PersonService.kt
package be.superjoran.mint.services

import be.superjoran.common.model.DomainObjectCrudService
import be.superjoran.mint.domain.Person
import be.superjoran.mint.domain.searchresults.PersonCandidate

interface PersonService : DomainObjectCrudService<Person> {
    fun logIn(person: PersonCandidate): Person

    fun create(personCandidate: PersonCandidate): Person
}
